examples/experimental/koen/Sy Resnet.ipynb

```diff
@@ -0,0 +1,530 @@
+{
```

Comment:
<p>glad to se this is working without model.send() although i'm not sure where the model or its parameters are being sent to the remote machine. where is this happening?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5455/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>both, because model is an argument to the plan it is sent to the plan vm automatically. model is a symodule and therefore uses the object2proto in lib/torch/module line 419</p>

Reply:
<p>ahhhhh!</p>